### ISSUE TITLE
Dimension arrows aren't showing up consistently

### ISSUE THREAD

**Issue by vvvo** [NYCPlanning/labs-applicantmaps#580]
See link: [https://applicantmaps.planning.nyc.gov/projects/jSeQAnGJ3HQz1+4HZFto/edit/map/edit](url)

Dimension arrows don't show up (not sure if it's because of the scale of the map). We might want to bump up it's size so the arrow is legible.

**Comment by allthesignals** [comment]
Thanks @vvvo — what do you mean this part of it?

<img width="482" alt="Screen Shot 2020-03-05 at 1 03 37 AM" src="https://user-images.githubusercontent.com/5004319/75952620-35843080-5e7d-11ea-820c-393b8c7c918d.png">

Not in the print view itself?

**Comment by vvvo** [comment]
Yep - the arrow heads don't appear when you go into the print view.
